libfcgi/os_unix.c

```diff
@@ -110,11 +110,13 @@ void OS_ShutdownPending()
 
 static void OS_Sigusr1Handler(int signo)
 {
+    (void)signo;
```

Comment:
The attribute [[maybe_unused]] shall be standard in c2x, which may be expected in 2021. I have it with clang 9.0.1 and -std=c2x